defaults/main.yml

```diff
@@ -46,6 +46,15 @@ etckeeper_lowlevel_package_manager: |
 #
 etckeeper_vcs: 'git'
 
+# .. envvar:: etckeeper_vcs_username
+#
+# username to use when making VCS commits
+etckeeper_vcs_username: "{{ansible_hostname}}"
```

Comment:
Just a convention thing. Can you change that to `etckeeper_vcs_username: '{{ ansible_hostname }}'` maybe? This makes it easier to copy paste stuff. When you need to quote inside Jinja, we use `"` for this for example: `etckeeper_vcs_username: '{{ ansible_hostname + "-staging" }}'`.